### PR TITLE
Disable spring sleuth analyzer by default

### PR DESCRIPTION
**Pull request by mrproliu**
# Ref #7091 
- Disable spring sleuth analyzer by default
- Add `agent-analyzer.meterAnalyzerActiveFiles` config description in the document.
- Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Release Notes.
 
 #### Java Agent
 * Supports modifying span attributes in async mode.
+* Disable Spring sleuth meter analyzer by default.
```
Comment (wu-sheng) [inline CHANGES.md:12]: This is backend change.

File: docs/en/setup/backend/backend-receivers.md [hunk 2]
```diff
@@ -148,6 +148,22 @@ receiver-meter:
...
 
+To activate the meter rule files:
+```yaml
+agent-analyzer:
+  selector: ${SW_AGENT_ANALYZER:default}
+  default:
+    meterAnalyzerActiveFiles: ${SW_METER_ANALYZER_ACTIVE_FILES:} # Which files could be meter analyzed, files split by ","
+```
+
+The receiver adds labels with `key = service` and `key = instance` to the collected data samples,
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-receivers.md:159]: Is meter system doing this?

File: docs/en/setup/backend/backend-receivers.md [hunk 3]
```diff
@@ -148,6 +148,22 @@ receiver-meter:
...
+and values from service and service instance name defined in SkyWalking Agent,
+for identification of the metric data.
+
+| Rule Name | Description | Configuration File | Data Source |
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-receivers.md:163]: I think this table belongs to `backend-meter.md`? As this config not belonging to the receiver.

File: docs/en/setup/backend/backend-receivers.md (hunk 2 above)
Comment (mrproliu) [reply docs/en/setup/backend/backend-receivers.md:159]: The agent analyzer analyzes the protocol and adds these two keys as labels. After this, the `MAL` could build the entity from these labels.

File: docs/en/setup/backend/backend-receivers.md (hunk 3 above)
Comment (mrproliu) [reply docs/en/setup/backend/backend-receivers.md]: Changed to the `backend-meter.md`

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**